lib/matplotlib/ticker.py

```diff
@@ -1170,25 +1170,200 @@ class LogitFormatter(Formatter):
     """
     Probability formatter (using Math text).
     """
-    def __call__(self, x, pos=None):
-        s = ''
-        if 0.01 <= x <= 0.99:
-            s = '{:.2f}'.format(x)
-        elif x < 0.01:
-            if is_decade(x):
-                s = '$10^{{{:.0f}}}$'.format(np.log10(x))
-            else:
-                s = '${:.5f}$'.format(x)
-        else:  # x > 0.99
-            if is_decade(1-x):
-                s = '$1-10^{{{:.0f}}}$'.format(np.log10(1-x))
+
+    def __init__(
+        self,
+        *,
+        use_overline=False,
+        one_half=r"\frac{1}{2}",
+        minor=False,
+        minor_threshold=25,
+        minor_number=6,
+    ):
+        r"""
+        Parameters
+        ----------
+        use_overline : bool (default: False)
+            if x > 1/2, with x = 1-v, indicate if x should be displayed as
+            $\overline{v}$. The default is to display $1-v$.
+
+        one_half : str (default: r"\frac{1}{2}")
+            the string used to represent 1/2.
+
+        minor : bool (default: False)
+            indicate if the formatter is formatting minor ticks or not.
+            Basically minor ticks are not labelled, except when only few ticks
+            are provided, the most espaced ticks are labelled. See others
```

Comment:
There's a typo here, but not sure what "most espaced" should be replaced with (as "most spaced" is also not clear to me.)

Also, others -> other

Reply:
I changed the `others`.

I reword the spacing between ticks.

I keep this thread open, I am not sure about my words.

Reply:
You are right for the idea, but it is not what the code is doing. The formatter for minor ticks only know the location for minor ticks and doesn't know the location of major ticks, and only minor ticks is used.

Due to behavior on LogitLocator, the only way to having few (<3) major ticks is equivalent to have a few (<25) minor ticks. So the code apply the threshold on the number of minor ticks.

Same idea for spacing, from the space between minor ticks and the LogitLocator behavior, making the decision from space between minor ticks is sufficient to have a nice behavior.

Therefore I think keeping the name ticks without qualifier is a solution. The other solution, is to use the minor qualifier all time and add another comment explaining why only minor ticks are used.